modules/ROOT/pages/installation.adoc

```diff
@@ -77,4 +77,15 @@ with partitions for `/boot`, `/`, `swap` and `/var/home`.
 
 Manual partitioning on Silverblue can be done with `Btrfs`,
 https://en.wikipedia.org/wiki/Logical_Volume_Manager_%28Linux%29[LVM], as well 
-as standard partitions or an `xfs` filesystem.  
+as standard partitions or an `xfs` filesystem.
+
+[[manual-partition-multiboot]]
+=== Manual Partitioning in a "multiboot" scheme
+
+Some users will want to install Silverblue alongside other operating systems -- including or *NIX flavours. For them we provide a minimal, functional partition scheme, consisting of:
+
+1. `/boot/efi` on a physical partition -- typically the active ESP partition shared among bootloaders
+2. `/boot` on a physical, freshly created partition, preferably formatted to `btrfs`
```

Comment:
Does OSTree support that? I'm not mistaken the BLS snippets generated by OSTree have some assumptions (i.e: expects that there's a boot partition and doesn't work if `/boot` is not a mount point). So I wonder if this would work if for example a user has a `btrfs` subvolume in a `btrfs` boot partition.

I would say that's safer to use `xfs` or `ext4` for the boot partition, as is used in the non-ostree Fedora Workstation variants.